### DISCUSSION TITLE
Compiled JARs

### DISCUSSION THREAD

**Discussion by sergeysedoy97** [javamelody/javamelody#1171] [Q&A] [answered]
Is there a way to include JavaMelody in compiled application?

**Comment by evernat** [comment, marked as answer]
Yes. It is explained at https://github.com/javamelody/javamelody/wiki/UserGuide#1-jar-files

In the case that you have a compiled application such as a webapp in war : 
Copy the files [javamelody.jar](https://github.com/javamelody/javamelody/releases) and [jrobin-1.5.9.jar](https://repo1.maven.org/maven2/org/jrobin/jrobin/1.5.9/jrobin-1.5.9.jar) to the WEB-INF/lib directory of the war of the webapp to monitor.
